src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-codesonarProjectFile.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    Path to the CodeSonar project file which was used when invoking the codesonar command on the build.

```

Comment:
In this case, we must assume that the user has already done the build/analysis and has created the .prj and .prj_files entries.  This field gives the user the ability to tell the plugin where to find the prj_files.  The current text says:

>    Path to the CodeSonar project file which was used when invoking the codesonar command on the build.
>
>    The path should be relative to the Jenkins build working directory.

I think it is best to avoid specifics about what exactly is in the .prj or .prj_files and try to prompt the user to provide whatever path they used when they invoked `codesonar analyze` - which I think will typically not have an extension at all.  The plugin can handle either .prj or .prj_files extensions, but I am not sure that we need to point that out specifically.  I suggest the following text:

> CodeSonar project file path.
>
> This should match the path to the project file used in the invocation of `codesonar analyze` in your Jenkins build.  The path should be relative to the Jenkins build working directory.

Reply:
If they've already done the analysis then the needs are slightly different - if they haven't build/analyzed, I _would_ want to clearly point out that a directory will be created and populated, but if analysis is already done then I agree it's unnecessary.

"CodeSonar project file path.

This should match the project file path you specified in your `codesonar build` and `codesonar analyze` command lines.
Relative paths are intterpreted with respect to the Jenkins build working directory.
"